build/index.js

```diff
@@ -321,13 +321,19 @@ async function buildLiveSamplePageFromURL(url) {
   throw new Error(`No live-sample "${sampleID}" found within ${documentURL}`);
 }
 
+// This is used by the builder (yarn build) and by the server (JIT).
+function renderContributorsTxt(names) {
+  return `# Contributors to this:\n${names.join("\n")}\n`;
```

Comment:
Maybe this should also contain something like:
```
+= `\nFor contributors on GitHub see https://github.com/mdn/content/commits/main/${folder}/index.html`
```

Reply:
@peterbe Three thoughts:
1. I think it's essential to include a link to our GitHub contributors as well.
1. I think we should use `Original Wiki Contributors` (or something like that) instead of `Contributors`.
1. I think we should make the link to the GitHub contributors appear first, and then list the original Wiki contributors.

Reply:
I'm on it.